src/main/java/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -494,7 +518,7 @@ public void addNotification(int accountId, int chatId, int msgId) {
                             lines = new ArrayList<>();
                             accountInbox.put(chatId, lines);
                         }
-                        lines.add(line);
+                        lines.add(tickerLine);
```

Comment:
I think there is a misunderstanding, I wanted to suggest changing `lines.add(tickerLine);` to `lines.add(shortLine);` in order not to have the redundant `Bob: ` prefix

Reply:
> but, most importantly for this PR: this is offtopic, as far as i see, things are unchanged here, at least this was the gist :)

As far as I understood the code, `line` was renamed to `shortLine` and `tickerLine` is still called `tickerLine` throughout the code? So, to get the behavior of the old code, `line` should be renamed to `shortLine` also here.

However, I just see that there was a bug in the old code:

```java
            // if privacy allows, for better accessibility,
            // prepend the sender in the ticker also for one-to-one chats (for group-chats, this is already done)
            String tickerLine = line;
            if (!dcChat.isMultiUser() && privacy.isDisplayContact()) {
                line = dcMsg.getSenderName(dcContext.getContact(dcMsg.getFromId()), false) + ": " + line;
            }
```
here, `line` is changed instead of `tickerLine`, even though the comment says that the sender should be prepended to the ticker. 

This bug is gone now, and it looks like the old behavior is produced again by replacing `line` with `tickerLine` here.

Reply:
let's maybe continue below on code-level, not sure if i could explain it better there :) (or if i am wrong at the end :)